dbt_subprojects/hourly_spellbook/macros/blockchain_transaction_address_metrics.sql

```diff
@@ -0,0 +1,109 @@
+{% macro blockchain_transaction_address_metrics(blockchain) %}
+
+with contracts as (
+    select
+        distinct address
+    from
+        {{ source(blockchain, 'creation_traces') }}
+    where
+        1 = 1
+        and block_time >= timestamp '2024-08-01'
+        {% if is_incremental() %}
+        and {{ incremental_predicate('block_time') }}
+        {% endif %}
```

Comment:
we should not filter this incrementally, as a contract could have been deployed at any moment in the past.

Reply:
good call, likely wrote properly on historical then updated to incorrect value on incremental with filter. i have removed.